info/kubelessInfo.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+const _ = require('lodash');
+const Api = require('kubernetes-client');
+const BbPromise = require('bluebird');
+const chalk = require('chalk');
+const helpers = require('../lib/helpers');
+
+class KubelessInfo {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options || {};
+    this.provider = this.serverless.getProvider('google');
+    this.commands = {
+      info: {
+        usage: 'Display information about the current functions',
+        lifecycleEvents: [
+          'info',
+        ],
+        options: {
+          verbose: {
+            usage: 'Display metadata',
+            shortcut: 'v',
+          },
+        },
+      },
+    };
+    this.hooks = {
+      'info:info': () => BbPromise.bind(this)
+        .then(this.validate)
+        .then(this.infoFunction),
+    };
+  }
+
+  validate() {
+    helpers.validateEnv();
+    const unsupportedOptions = ['stage', 'region'];
+    helpers.warnUnsupportedOptions(
+      unsupportedOptions,
+      this.options,
+      this.serverless.cli.log.bind(this.serverless.cli)
+    );
+    return BbPromise.resolve();
+  }
+
+  formatMessage(service, functions, options) {
+    if (options && !options.color) chalk.enabled = false;
+    let message = '';
+    message += `\n${chalk.yellow.underline('Service Information')}\n`;
```

Comment:
yes, I took it from the AWS plugin:
https://github.com/serverless/serverless/blob/9f7e740b09ac571ecadf6c3d0a7d0a87c5c15132/lib/plugins/aws/info/display.js#L10